packages/dev-tools/package.json

```
@@ -3,14 +3,14 @@
   "version": "0.10.29",
   "main": "build/server/index.js",
   "scripts": {
-    "dev": "NODE_ENV=development ts-node ./server/dev-server",
+    "dev-server": "NODE_ENV=development nodemon --watch server -e js,ts --exec \"ts-node ./server/dev-server\"",
+    "dev-ui": "cd client && expo start --web-only",
```

Possibly just start expo web directly via xdl or use the bin
```suggestion
    "dev-ui": "cd client && ../expo-cli/bin/expo.js start --web-only",
```